pkgs/applications/audio/mopidy/mopidy.nix

```diff
@@ -50,25 +50,26 @@ pythonPackages.buildPythonApplication rec {
         }
       ))
     ]
-    ++ lib.optional (!stdenv.hostPlatform.isDarwin) pipewire;
-
-  propagatedBuildInputs =
-    [ gobject-introspection ]
-    ++ (
-      with pythonPackages;
-      [
-        gst-python
-        pygobject3
-        pykka
-        requests
-        setuptools
-        tornado
-      ]
-      ++ lib.optional (!stdenv.hostPlatform.isDarwin) dbus-python
-    );
+    ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ pipewire ];
 
   propagatedNativeBuildInputs = [ gobject-introspection ];
 
+  propagatedBuildInputs = [ gobject-introspection ];
+
+  build-system = [ pythonPackages.setuptools ];
+
+  dependencies =
+    with pythonPackages;
+    [
+      gst-python
+      pygobject3
+      pykka
+      requests
+      setuptools
+      tornado
+    ]
+    ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ dbus-python ];
```

Comment:
I swapped `lib.optional` for `lib.optionals`.
I can revert this, but I find it more readable as `lib.optionals`